### PR TITLE
DT-2441 Improve CSS, breadcrumbs, filter form

### PR DESCRIPTION
**Pull request by andymarke**
3 unrelated improvements to previous PR
* Fix CSS to remove moj filter hack - ensure <p> are styled correctly with the govuk global flag
* Fix need for JavaScript to submit filter form - move form up a level
* Improved breadcrumb handling on edit user - breadcrumbs differ when navigating from the combined filter page

### CODE REVIEW

File: backend/controllers/search.js [hunk 1]
```diff
@@ -46,6 +46,8 @@ const searchFactory = (
 
     // stash away the search url in the session to provide in breadcrumbs to go back
     req.session.searchResultsUrl = req.originalUrl
+    delete req.session.searchTitle
+    delete req.session.searchUrl
```
Comment (andymarke) [inline backend/controllers/search.js:51]: remove breadcrumb stuff placed by the new search filter page

File: integration-tests/support/dpsuser.helpers.js [hunk 2]
```diff
@@ -43,13 +43,15 @@ export const goToSearchWithFilterPage = ({ isAdmin = true, totalElements = 21, s
...
-export const editUser = ({ isAdmin = false, nextPage }) => {
-  const results = goToResultsPage({ isAdmin, nextPage })
-
+export const editUser = ({ isAdmin = false, fromSearchFilterPage = false, nextPage }) => {
   cy.task('stubDpsUserDetails')
   cy.task(isAdmin ? 'stubDpsUserGetAdminRoles' : 'stubDpsUserGetRoles')
   cy.task('stubEmail', { email: 'ITAG_USER@gov.uk' })
 
-  results.edit('ITAG_USER5')
+  if (fromSearchFilterPage) {
```
Comment (andymarke) [inline integration-tests/support/dpsuser.helpers.js:51]: allow navigating to edit user via both versions of the search page

File: sass/application.scss [hunk 3]
```diff
@@ -1,4 +1,5 @@
 $govuk-page-width: 1170px;
+$govuk-global-styles: true;
 @import 'node_modules/govuk-frontend/govuk/all';
```
Comment (andymarke) [inline sass/application.scss:3]: inline with typescript template; this ensures <p> etc are styled the GDS way

File: views/searchWithFilter.njk [hunk 4]
```diff
@@ -132,7 +130,9 @@
       <div class="govuk-grid-column-full">
         <div class="moj-filter-layout">
         <div class="moj-filter-layout__filter">
+        <form id="filter-form" action="{{ searchUrl }}" novalidate>
           {{ mojFilter(currentFilter | toUserSearchFilter(groupOrPrisonDropdownValues, roleDropdownValues, filterOptionsHtml, showGroupOrPrisonDropdown))}}
+        </form>
         </div>
```
Comment (andymarke) [inline views/searchWithFilter.njk:136]: move form so submit button is now inside form

File: views/userDetails.njk [hunk 5]
```diff
@@ -5,14 +5,28 @@
...
  {% block beforeContent %}
+      {% if searchUrl === searchResultsUrl %}
+      {% set breadcrumbs = 
+        [
+          { text: "Home", href: homeUrl },
+          { text: "Manage user accounts", href: '/' },
+          { text: searchTitle, href: searchUrl },
+          { text: staff.name}
+        ]
+       %}
```
Comment (andymarke) [inline views/userDetails.njk:16]: need one less breadcrumb for the new search since results and filter are on the same page

**Review by mikehalmamoj — APPROVED** [review]

**Pull request merged**